docs/getting-started-guides/scratch.md

```diff
@@ -652,7 +655,7 @@ This pod mounts several node file system directories using the  `hostPath` volum
 
 - The `/etc/ssl` mount allows the apiserver to find the SSL root certs so it can
   authenticate external services, such as a cloud provider.
-  - This is not required if you do not use a cloud provider (e.g. bare-metal).
+  - This is not required if you do not use a cloud provider (for example, bare-metal).
```

Comment:
Another where the "for example" would make more sense after the example:

    cloud provider (bare-metal for example)